bootstrap/sql/mysql/create_tables.sql

```diff
@@ -69,6 +69,14 @@ CREATE TABLE IF NOT EXISTS file (
     UNIQUE KEY `jars_UK_name_version` (name, version)
 );
 
+CREATE TABLE IF NOT EXISTS fileblob (
+    name VARCHAR(255) NOT NULL,
+    version BIGINT NOT NULL,
+    data LONGBLOB NOT NULL,
+    timestamp BIGINT,
```

Comment:
"timestamp" is a keyword but not a reserved keyword, so it can be used as column name without quoting. Actually `timestamp` is used in most of the streamline tables so if we want to rename we can take it up in a separate issue, but its safe to leave it as is.
https://dev.mysql.com/doc/refman/5.7/en/keywords.html